client/ionic/src/pages/About.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { IonContent, IonPage, IonSlides, IonSlide, IonCard, IonCardContent, IonButton } from '@ionic/react';
+import styles from './Tab1.module.css'
```

Comment:
is this the right css file?  if we're reusing the css should the modules be refactored to share the common styles, or just rename Tab1 to indicate its shared usage?

Reply:
nvm - draft status

Reply:
yes.  arent you supposed to be asleep? =)

Reply:
just finished dinner and i'm a workaholic ;)

Reply:
I don't have any idea what that is ;)

Reply:
FYI, I don't think we use tab1 and tab2 anymore.  can probably remove them as well and rename css to common.css or whatever you suggest.

Reply:
lol! btw, if we're sticking with Ionic React PWA in the long-term, do we want to consider using a different system to manage styles, such as [`styled-components`](https://github.com/styled-components/styled-components)? not a decision we would probably make for v0.1, just brain dumping

Reply:
CSS modules is meant for styling components. (I chose it because it is the default)

Ionic provides guidance on global styles as well, they should go into themes.

Reply:
removed this and the unused pages and css